tldw_Server_API/app/core/Sharing/clone_service.py

```
@@ -0,0 +1,210 @@
+"""Service for cloning shared workspaces into the accessor's own DB."""
+from __future__ import annotations
+
+import uuid
+from typing import Any, Callable
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)